.packaging/deb/control

```diff
@@ -0,0 +1,15 @@
+Source: ponyc
+Maintainer: Pony Core Team <buildbot@pony.groups.io>
+Build-Depends: debhelper (>= 9.0.0), git, zlib1g-dev, libncurses5-dev, libssl-dev, libpcre2-dev, llvm-3.9, llvm-3.9-dev
```

Comment:
Do we actually need libncurses5-dev, libssl-dev, libpcre2-dev during build?

Reply:
@mfelsche yes, because we run tests as part of the build process before packaging to ensure that the resulting packages will work as expected.

Reply:
to clarify, by "build process", i mean the deb and rpm build processes on copr/launchpad.